catkin-tools-setup/install.yaml

```diff
@@ -0,0 +1,2 @@
+- type: ppa
+  name: ppa:catkin-tools/ppa
```

Comment:
Is this really less intrusive than installing `python3-osrf-pycommon` extra?

Both are a workaround. But this one cannot be reverted. So in the future when it **might** be updated officially, we're always stuck on this (then unmaintained) ppa.

Reply:
No, this is to get updated to the newest version `>0.7.0`.

The newest version will probably not be released on `packages.ros` (see https://github.com/catkin/catkin_tools/issues/594#issuecomment-883707273). I don't think we should be using the old `0.5.X` version.

The dependencies of the newest version are fixed, so there is no need anymore to install `python3-osrf-pycommon`.

When this ppa get unmaintained in the future, we just need to write a bash script to remove the ppa.

Reply:
Here ya go, this can be reverted again ;-)
The fix is released.

Reply:
See https://github.com/tue-robotics/tue-env-targets/pull/241